### PR TITLE
Add entry on PHP String Functions echo()

### PR DESCRIPTION
**Pull request by tokyohmachine**
### Description

The entry was revised.

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/echo/echo.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+```php
+  $name = "John";
+  echo "My name is $name";
+
+
+  echo 'My name is '.$name;
+```
+
+Output:
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/echo/echo.md:56]:
```suggestion
This example will output:
```

**Comment by tokyohmachine** [comment]
Hi,

I have made the requested changes in response to your feedback.

File: content/php/concepts/string-functions/terms/echo/echo.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Output'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/web-development'
+---
+
+> The metadata above provides information about the entry for the PHP String Functions echo().
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/echo/echo.md:15]:
```suggestion
```
Remove this line

File: content/php/concepts/string-functions/terms/echo/echo.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/echo/echo.md:16]: Remove extra line break.

File: content/php/concepts/string-functions/terms/echo/echo.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+One of the most commonly used functions in PHP is **`echo()`**, which is used to output one or more strings to the browser or command-line interface.
+## Syntax:
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/echo/echo.md:18]:
```suggestion

## Syntax:
```
Add line break before header.

File: content/php/concepts/string-functions/terms/echo/echo.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  $name = "John";
+  echo "My name is $name";
+  or
+  echo 'My name is ' . $name;
+```
+
+This example will output:
+
+```shell
+My name is John
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/echo/echo.md:47]:
```suggestion
My name is John
My name is John
```
The example prints this twice.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@tokyohmachine a few more comments.

File: content/php/concepts/string-functions/terms/echo/echo.md (hunk 3 above)
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/echo/echo.md:16]:
```suggestion
```
Remove extra line break

Comment (tokyohmachine) [reply content/php/concepts/string-functions/terms/echo/echo.md]: Hello! I have made the requested changes to address the feedback provided.  Thank you!

**Review by KTom101 — CHANGES_REQUESTED** [review]
@tokyohmachine I've added a second review for this entry.

**Comment by KTom101** [comment]
@tokyohmachine I'm just checking in to see how's it going with the changes.

**Comment by tokyohmachine** [comment]
Hello, I make the changes requested.

**Comment by SSwiniarski** [comment]
@tokyohmachine, I made a few edits and ran `npm run format` so that all the checks pass. That way this can be merged as soon as @KTom101 reviews and approves your edits.

File: content/php/concepts/string-functions/terms/echo/echo.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+```php
+echo '<h1>Welcome to my website!</h1>';
+```
+
+This will output a heading tag to the screen.
+
+> **Note:** HTML and PHP code can both be used within `echo()` statements.
+
+One important thing to keep in mind when using `echo()` is that it does not return a value. In other words, the output of `echo()` cannot be assigned to a variable or used in an expression.
```
Comment (KTom101) [inline content/php/concepts/string-functions/terms/echo/echo.md:53]:
```suggestion
One important thing to keep in mind when using `echo()` is that it does not return a value. In other words, the output of `echo()` cannot be assigned to a variable or used in an expression.

```

File: content/php/concepts/string-functions/terms/echo/echo.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+`echo()` can also be used to output variables:
+
+```php
+$name = "John";
+echo "My name is $name";
+```
+
+In the example above, the output will be "My name is John".
+The concatenation operator (.) joins the string "My name is " with the variable `$name`.
```
Comment (KTom101) [inline content/php/concepts/string-functions/terms/echo/echo.md:42]:
```suggestion
The concatenation operator (.) joins the string "My name is " with the variable `$name`.

```

**Review by KTom101 — APPROVED** [review]
@tokyohmachine Looks good, I just added a couple more line breaks.

**Comment by KTom101** [comment]
@tokyohmachine The entry is live now: https://github.com/tokyohmachine/code-academy-project.git 🎉

Currently, you're listed as an anonymous contributor:

Screenshot 2023-06-07 at 7 49 47 PM

If you want to remain anonymous, there is nothing more you need to do. If you wish to be listed as a contributor, make sure you've added your GitHub username to your Codecademy profile using the following steps:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

YourGitHubName

2.) Then log out and log back into Codecademy using your GitHub login:

ReLogIn

> **Note:** Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to work properly. If you use the same email for both GitHub and Codecademy already, then you're set.

cc @yangc95 @SSwiniarski

**Pull request merged**